### ISSUE TITLE
Inputs without labels

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#7565]
### Problem description

In general, most input fields lack label-texts. Placeholders are used instead, but since that text disappears as soon as the user starts to type in the field this is not enough to comply with this success criterion.

There are also contrast issues with placeholder texts, and when improving contrasts it can make the user think the field is already filled in. So avoiding placeholder texts, in general, tends to be a good idea.

### Talk app

The search field in the Talk app does not have a visual label, only a placeholder text.

Every input field must have a visible label text. An exception is often made for search fields, but when it’s not a standard search-function, like here where the search is for conversations and users, a visible label is needed to clarify this.

It’s also worth noting that even though search fields are often excepted, no such exception exists in the WCAG- or EN 301 549-standards.

**Comment by nickvergessen** [comment]
Will be fixed by using the new input component of the Vue lib
- [ ] https://github.com/nextcloud/nextcloud-vue/pull/2868
